src/canopy-webpack-config.js

```diff
@@ -2,13 +2,12 @@ const path = require('path')
 const { CleanWebpackPlugin } = require('clean-webpack-plugin')
 const merge = require('webpack-merge')
 const BundleAnalyzerPlugin = require('webpack-bundle-analyzer').BundleAnalyzerPlugin
-const {UnusedFilesWebpackPlugin} = require('unused-files-webpack-plugin')
 const fs = require("fs")
 const homedir = require("os").homedir()
 
 let isDevServer = false
 
-if (process.argv.some(arg => arg.includes('webpack-dev-server'))) {
+if (process.argv.some(arg => arg.includes('serve'))) {
```

Comment:
webpack 4 dev-server cli has changed it's main serve command from `webpack-dev-server` to `webpack serve`